tools/metadata_loader/main.py

```diff
@@ -0,0 +1,185 @@
+import argparse
+import json
+import os
+from urllib.parse import quote
+
+import requests
+from azure.storage.blob import (  # pyright: ignore[reportMissingImports]
+    BlobServiceClient,
+)
+from dotenv import load_dotenv
+
+
+def get_config():
+    load_dotenv()
```

Comment:
Should we not put load dotenv to the begining of the file like line 11 ? doesnt need to be inside the function.

Reply:
ok